### ISSUE TITLE
Long sentences in Skills section going out of bounds of the page

### ISSUE THREAD

**Issue by 7h3-10n3-r4n63r** [saadq/resumake.io#127]
**Describe the bug**
In the Skills section I have noticed two different things:
1. If the Skill Name is too long it's not wrapping after certain limit
2. If the Skill Detail is too long it's not wrapping after the page margin is reached and is going out of bounds
I have noticed this bug in the templates 1, 2, 4, 5

I have tested this in all other fields like Education, Work, Projects and Awards, but this bug exists only in Skills Section

**To Reproduce**
1. Go to 'https://resumake.io/generator/templates'
2. Select any one of the following templates: 1, 2, 4, 5
3. Select the 'Skills' Section
4. Enter a long Skill Name and Skill Details
5. See error

**Expected behavior**
Long Skill Names and Skill Details should be wrapped after certain limit

**Screenshots**
![Skill_name_Bug](https://user-images.githubusercontent.com/40575581/74702940-a1d22380-5231-11ea-9ba9-0be92fe4b7f8.png)

**Comment by nivabecassis** [comment]
A workaround is to create a new skill section with an empty skill name. The skills are on a new line and are still properly aligned.

**Comment by devkdn** [comment]
Using an empty skill name adds Misc for me
![image](https://user-images.githubusercontent.com/22102599/122645765-991d8600-d0e1-11eb-8834-3f51965fc945.png)

**Comment by lukew3** [comment]
> Using an empty skill name adds Misc for me

Yeah, in some templates it defaults to adding `Misc:`. I found that using a `.` (period) is better than leaving it empty as it will show something that looks like `.:` instead.
![Screenshot from 2022-07-05 23-27-38](https://user-images.githubusercontent.com/47042841/177461883-80be7eca-ffaf-427e-b129-cd10db27b4af.png)
